kubespawner/__init__.py

```diff
@@ -11,8 +11,11 @@
 # We export KubeSpawner specifically here. This simplifies import for users.
 # Users can simply import kubespawner.KubeSpawner in their applications
 # instead of the more verbose import kubespawner.spawner.KubeSpawner.
+#
+# We're also exporting the convenience class MultiNamespaceKubeSpawner,
+# for the same reason.
 
-from kubespawner.spawner import KubeSpawner
+from kubespawner.spawner import KubeSpawner, MultiNamespaceKubeSpawner
```

Comment:
Given how MultiNamespaceKubeSpawner is implemented, I think we can lose the additional class and instead enable this behavior with:

```python
c.KubeSpawner.enable_user_namespaces = True
```

Reply:
Done!